lang/python/setup_pip.py

```diff
@@ -217,18 +217,15 @@ def get_library_dirs():
 # Here's the configure/make operations we perform before the python extension
 # is linked.
 configure_cmds = [
-    'cmake -B cmake_pip_build -G Ninja -DCMAKE_C_FLAGS="${CFLAGS:-}" -DENABLE_PYTHON=1 ' + \
+    'cmake -B cmake_pip_build -G Ninja -DENABLE_STATIC=1 -DCMAKE_C_FLAGS="${CFLAGS:-}" -DENABLE_PYTHON=1 ' + \
     ' '.join(map(lambda name: '-DHAVE_BUILTIN_EXTENSION_' + name.upper() + '=1', builtin_names)),
 ]
 
 # build all the builtins, at the moment they are all compressors.
 make_cmds = []
 for name in builtin_names:
     make_cmds.append('ninja -C ' + build_dir  +  ' ext/compressors/' + name + '/all')
-if sys.platform == 'darwin':
-    make_cmds.append('ninja -C ' + build_dir + ' libwiredtiger.dylib')
-else:
-    make_cmds.append('ninja -C ' + build_dir + ' libwiredtiger.so')
+make_cmds.append('ninja -C ' + build_dir + ' libwiredtiger.a')
 make_cmds.append('ninja -C ' + build_dir + ' lang/python/all')
```

Comment:
@alisonfel Do we still need this line?

Reply:
Yep, we still need this line. We need to invoke the python build to generate the SWIG files, which in turn are needed to compile the python library.